conf/messages

```diff
@@ -94,12 +94,12 @@ addressLookup.confirmPage.title                                 = Confirm busine
 addressLookup.confirmPage.heading                               = Confirm business address
 
 # Agent Business start date
-business.agent.start_date.title                                 = When did your client‘s business start trading?
-business.agent.start_date.heading                               = When did your client‘s business start trading?
+business.agent.start_date.title                                 = When did your client''s business start trading?
```

Comment:
https://design.tax.service.gov.uk/hmrc-content-style-guide/ says we should use curly apostrophes rather than the ones you are using here